src/tron.jl

```diff
@@ -130,7 +176,7 @@ function solve!(
 
   αC = one(T)
   tr = TRONTrustRegion(gt, min(max(one(T), πx / 10), 100))
-  @info log_header(
+  (verbose > 0) && @info log_header(
```

Comment:
```suggestion
  verbose > 0 && @info log_header(
```

Reply:
The parenthesis are following the formatting.

Reply:
JuliaFormatter?

Reply:
I rechecked and you are right, we can remove the parenthesis. Thanks!